deepprofiler/dataset/illumination_statistics.py

```diff
@@ -8,7 +8,7 @@
 
 
 def illum_stats_filename(output_dir, plate_name):
-    return output_dir + "/" + plate_name + "/intensities/" + plate_name + ".pkl"
+    return output_dir + "/" + plate_name + "/" + plate_name + ".pkl"
```

Comment:
String formatting can be used, ex. `"{}/{}/{}.pkl".format(output_dir, plate_name, plate_name)`